swix/packages.config

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MicroBuild.Core" version="0.2.0" targetFramework="net46" developmentDependency="true" />
+  <package id="Nerdbank.GitVersioning" version="1.4.30" targetFramework="net46" developmentDependency="true" />
```

Comment:
Ensure we have this registered for internal usage.

Reply:
I believe we should, many other packages are using it. I'll double check to be sure.

Reply:
Approved!